WordPress/WordPressTest/WPUserAgentTests.m

```diff
@@ -0,0 +1,67 @@
+//
+//  WPUserAgentTests.m
+//  WordPress
+//
+//  Created by Diego E. Rey Mendez on 3/17/15.
+//  Copyright (c) 2015 WordPress. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "WPUserAgent.h"
+
+static NSString* const WPUserAgentKeyUserAgent = @"UserAgent";
```

Comment:
Part of my idea of not using the App constants in the unit tests is to make sure they don't change.  Even though this seems like duplication it's actually part of the checks I want to enforce (imagine a user changing a key by mistake and saving it).

Makes sense?